install/install_utils.php

```diff
@@ -982,23 +984,34 @@ function handleHtaccess()
     $restrict_str .= <<<EOQ
 RedirectMatch 403 {$ignoreCase}.*\.log$
 RedirectMatch 403 {$ignoreCase}/+not_imported_.*\.txt
-RedirectMatch 403 {$ignoreCase}/+(soap|cache|xtemplate|data|examples|include|log4php|metadata|modules)/+.*\.(php|tpl)
+RedirectMatch 403 {$ignoreCase}/+(custom/soap|custom/cache|custom/xtemplate|custom/data|custom/examples|custom/include|custom/log4php|custom/metadata|custom/modules|custom/vendor)/+.*\.(php|tpl)
```

Comment:
"custom/xtemplate custom/data custom/examples custom/log4php" etc, should it not be custom/application,custom/Extension etc, plus these are already covered by the next line anyway
is there maybe a regex  solotion to just inlucde the directories we want

Reply:
Yes, you're right, it's already covered by the next line. For the ```custom/application``` and ```custom/Extension``` I would think that those shouldn't be restricted.

Reply:
why not?

Reply:
My thinking was that we wouldn't want to restrict what you can do in custom beyond the default core restrictions. I wouldn't want to break someone's customisations in a minor release unnecessarily (Even if they are doing it the wrong way).